privacy/privacy-shield.md

```diff
@@ -1,10 +1,10 @@
 # Basecamp EU-US and Swiss-US Privacy Shield policy
 
-*Last updated: May 24, 2018*
+*Last updated: January 30, 2019*
 
 The privacy of your data—and it is your data, not ours!—is a big deal to us. We’ll only ever access your account to help you with a problem or squash a software bug. We’ll never open any uploaded files unless you ask us to. We log all access to all accounts by IP address, so we can always verify that no unauthorized access has happened for as long as the logs are kept.
 
-This policy applies to personal data from the European Union and from Switzerland that is collected, used, and retained by us in the United States.
+This policy applies to personal data from the European Union, the United Kingdom and from Switzerland that is collected, used, and retained by us in the United States.
```

Comment:
"from A, B and from C" → "from A, B, and C" or "from A, from B, and from C" ?

Reply:
```suggestion
This policy applies to personal data from the European Union, the United Kingdom, and Switzerland, which is collected, used, and retained by us in the United States.
```